docs/guides/authentication.md

```diff
@@ -4,4 +4,47 @@ title: Authentication
 
 # Authentication
 
-Documentation coming soon
+Authentication is the process of verifying that a user is "whom they claim to be" and
+should be handled by the framework you are using. Some already have a built-in
+authentication system (like Django); others, you have to provide it manually. It's not
+Strawberry's responsibility to authenticate the user, but it can provide some
+surroundings to the real authentication process. It's also very important to not confuse
```

Comment:
Unclear what "surroundings" means here

Reply:
Want so say that strawberry should be a wrapper around the real logic but it comed out badly... Will think to something better :)

Reply:
Hope now sounds better :)